### ISSUE TITLE
Backup of Phone contacts to sd card is not possible in  cm-11-20141006-UNOFFICIAL-LegacyXperia-haida

### ISSUE THREAD

**Issue by maruti40471** [LegacyXperia/local_manifests#835]
Backup of Phone contacts to sd card is not possible because after pressing import/export and then export to storage it indicates different contacts but after that there is no option to save these contacts.
Kindly solve this issue. 
   Thanks in advance.

**Comment by zweif** [comment]
When you check a checkbox, there should appear a hooklet left of the menu button (upper right corner). There is a check-all option behind the menu button too.

**Comment by maruti40471** [comment]
Of course there is a check-all option but after clicking the ✓ mark nothing happens.

**Comment by Grof87** [comment]
If you try to backup your contacts using Dialer -> Menu -> Import/Export for some reason it wont work. However if you do it using the People app it'll work without problems.

**Comment by mikeNG** [comment]
This might be fixed in today's build, I haven't tested it though

**Comment by maruti40471** [comment]
No, it is still not fixed. But thank you Grof87 for your alternative solution.Now I can backup the contacts using PEOPLE application. I Hope it will be rectified in future update.
    And lastly I would like to thank mikeNG for these fantastic updates for legacy devices. Can we expect LOLLIPOP update in near future?

**Comment by badboyasd** [comment]
yeah, some time back up can not complete, some time done.

**Comment by mikeNG** [comment]
Still present in latest build?

**Comment by zweif** [comment]
Yes, still doesn't work (using dialler).

**Comment by mikeNG** [comment]
Any logs? I don't have time to test :(
